addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -230,7 +230,8 @@ body.editor_enable {
 //s_dynamic_snippet
 body.editor_enable {
     .s_dynamic {
-        > * {
+        // All child selector doesn't target pseudo elements
+        > *, ::after, ::before {
```

Comment:
You inline comment does not help to understand in which case this is needed and the commit explanation is actually not right :wink: The problem is only due to the pointer-events property being explicitly changed by the stretched-link class :grimacing: 

![image](https://user-images.githubusercontent.com/10338094/181525571-5638f2da-6e0d-462e-bf8d-be48fe61310f.png)

You should thus fix that explicitly here by targeting that class only (not all ::after and ::before) so we know what this is about without thinking and even without comment (and of course change the commit explanation).